config.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config()
+
+const { Pool } = require( 'pg' )
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
`NODE_ENV` variable will need to be added to the `.env` file when we are ready for production (and of course change `ssl` to `isProduction`) 🏋️‍♂️